src/plot_erpgrid.jl

```diff
@@ -0,0 +1,87 @@
+"""
+    function plot_erpgrid!(f::Union{GridPosition, Figure}, data::Matrix{<:Real}, pos::Vector{Point{2,Float}}; kwargs...)
+    function plot_erpgrid(data::Matrix{<:Real}, pos::Vector{Point{2,Float}}; kwargs...)
+
+Plot an ERP image.
+## Arguments:
+- `f::Union{GridPosition, Figure}`: Figure or GridPosition that the plot should be drawn into;
+- `plotData::Matrix{<:Real}`: Data for the plot visualization;
+- `pos::Vector{Point{2,Float}}`: electrode positions.
+        
+## Keyword Arguments
+- `chanNum` (Number, `30`) - Number of channels.
+- `drawLabels` (bool, `false`) - Draw channels labels over each waveform. 
+- `times`: (Vector, `1:size(plotData, 2)`) - Vector of size()
+
+## Return Value:
+The input `f`
+"""
+
+# no figure?
+plot_erpgrid(plotData::Matrix{<:Real}, pos; kwargs...) =
+    plot_erpgrid!(Figure(), plotData, pos; kwargs...)
+
+function plot_erpgrid!(
+    f::Union{GridPosition,Figure},
+    plotData::Matrix{<:Real},
+    pos;
+    chanNum = 30,
+    drawLabels = false,
+    times = 1:size(plotData, 2),
+    kwargs...,
+)
+    plotData = plotData[1:chanNum, :]
+    pos = hcat([[p[1], p[2]] for p in pos]...)
+
+    pos = pos[:, 1:chanNum]
+    minmaxrange = (maximum(pos, dims = 2) - minimum(pos, dims = 2))
```

Comment:
we should move this standardization to an extra function. We need it in other places as well